jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/commands/SLATrackingCommand.java

```diff
@@ -100,11 +100,11 @@ public ExecutionResults execute(CommandContext ctx) throws Exception {
         parameters.put("now", new Date());
         StringBuilder lookupQuery = new StringBuilder();
 
-        lookupQuery.append("select log from NodeInstanceLog log where ");
-        lookupQuery.append("log.nodeInstanceId in ( select nil.nodeInstanceId from NodeInstanceLog nil where nil.slaDueDate < :now and nil.slaCompliance = 1 ");
+        lookupQuery.append("select log from NodeInstanceLog log, ProcessInstanceLog proc ");
+        lookupQuery.append("where log.nodeInstanceId in ( select nil.nodeInstanceId from NodeInstanceLog nil where nil.slaDueDate < :now and nil.slaCompliance = 1 ");
         lookupQuery.append("GROUP BY nil.nodeInstanceId ");
         lookupQuery.append("HAVING sum(nil.type) = 0) ");
-        lookupQuery.append("and log.type = 0 ");
+        lookupQuery.append("and log.type = 0 and proc.status NOT IN(2,3) and proc.processInstanceId = log.processInstanceId ");
```

Comment:
I believe better would be to use status = 1 as that is the only status that indicates active instance and I would move this join and condition part as the first ones, does not matter to look up node instance info if the process instance is not active